source/components/dateTimeStatic/dateTimeStatic.ts

```diff
@@ -24,20 +24,21 @@ export interface IDateTimeStaticController extends IDateTimeStaticBindings {
 }
 
 export class DateTimeStaticController implements IDateTimeStaticController {
-	dateValue: string;
+	dateValue: moment.Moment;
 	includeTime: boolean;
 	displayValue: string;
 	displayTimeZone: boolean;
 
 	static $inject: string[] = [__date.serviceName];
 	constructor(private dateUtility: __date.IDateUtility) {
-		this.displayValue = '';
 		if (this.dateValue != null && this.dateUtility.isDate(this.dateValue)) {
-			this.displayValue = moment(this.dateValue).format(__date.defaultFormats.dateFormat);
+			this.dateValue = moment(this.dateValue);
```

Comment:
Yes it is. moment() seems to function like $q.when() in that it's a way to verify that it's a moment.